curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/handle-click-events-with-javascript-using-the-onclick-property.english.md

```diff
@@ -103,7 +103,48 @@ tests:
 <section id='solution'>
 
 ```js
-// solution required
+<script>
+  document.addEventListener('DOMContentLoaded',function(){
+    // Add your code below this line
+    document.getElementById('getMessage').onclick=function(){}
+    // Add your code above this line
+  });
+</script>
+<style>
+  body {
+    text-align: center;
+    font-family: "Helvetica", sans-serif;
+  }
+  h1 {
+    font-size: 2em;
+    font-weight: bold;
+  }
+  .box {
+    border-radius: 5px;
+    background-color: #eee;
+    padding: 20px 5px;
+  }
+  button {
+    color: white;
+    background-color: #4791d0;
+    border-radius: 5px;
+    border: 1px solid #4791d0;
+    padding: 5px 10px 8px 10px;
+  }
+  button:hover {
+    background-color: #0F5897;
+    border: 1px solid #0F5897;
+  }
+</style>
+<h1>Cat Photo Finder</h1> 
+<p class="message box">
```

Comment:
```suggestion
<p class="message">
```
Sadly 'box' seems to have been forgotten for all of JSON APIs and Ajax.  It could be added back to all challenges, but it's out of scope for this PR.

Reply:
@ojeytonwilliams Can you create an issue for this?  If so, I will create a PR for the fix.

Reply:
marking as resolved, will be fixed with the other issue/PR